packages/vsrocq-language-server/vsrocq-language-server.2.3.2/opam

```diff
@@ -0,0 +1,53 @@
+opam-version: "2.0"
+maintainer: "Enrico Tassi <enrico.tassi@inria.fr>"
+authors: [ "Enrico Tassi" "Maxime Dénès" "Romain Tetley" ]
+license: "MIT"
+homepage: "https://github.com/rocq-prover/vsrocq"
+bug-reports: "https://github.com/rocq-prover/vsrocq/issues"
+dev-repo: "git+https://github.com/rocq-prover/vsrocq"
+
+build: [
+  [make "dune-files"]
+  [
+    "etc/rocq-wrap-coqc.sh" {!coq-core:installed}
+    "dune" "build" "-p" name "-j" jobs 
+  ]
+]
+depends: [
+  "ocaml" { >= "4.14" }
+  "dune" { >= "3.5" }
+  ("coq-core" { ((>= "8.18" < "8.21") | (= "dev")) }
+  | "rocq-core" { ((>= "9.0+rc1" < "9.2~") | (= "dev")) })
+  ("coq-stdlib" { ((>= "8.18" < "8.21") | (= "dev")) }
+  | "rocq-stdlib" { ((>= "9.0+rc1" < "9.2~") | (= "dev")) })
+  "yojson"
+  "jsonrpc" { >= "1.15"}
+  "ocamlfind"
+  "ppx_inline_test"
+  "ppx_assert"
+  "ppx_sexp_conv"
+  "ppx_deriving"
+  "sexplib"
+  "ppx_yojson_conv"
+  "ppx_import"
+  "ppx_optcomp"
+  "result" { >= "1.5" }
+  "lsp" { >= "1.15"}
+  "sel" {>= "0.6.0"}
+]
+conflicts: [
+    "vscoq-language-server" {< "2.7~"}
```

Comment:
Hm. I can easily imagine a new version conflicting with older ones, necessitating the `conflicts` entry. 
In the current setup however
`vscoq-language-server.2.3.2/opam` has
`depends: [ "vsrocq-language-server" {= version} ]`
yet that dependency is simultaneously conflicting IIUC.

I believe this is why all `vscoq-language-server.2.3.2` installations on the CI are failing with
`(failed: [SKIP] Package not available)`.

We should probably refine the constraints, but you folks know the conflicts better than us...

Reply:
@gares What is your take on this? 
`vscoq-language-server.2.3.2` is uninstallable with the current constraints and `conflicts`.
It doesn't make much sense to merge an uninstallable package, does it? :thinking:

Reply:
Indeed, check the last comment